dashboards/Experimental/Patroni_Details.json

```diff
@@ -0,0 +1,3766 @@
+          "query": "query_result(pg_settings_max_wal_size_bytes{service_name=~\"$service_name\"})",
+          "refId": "Metrics-max_wal_size-Variable-Query"
+        },
+        "refresh": 2,
+        "regex": "/ ([0-9\\.]+)/",
+        "skipUrlSync": false,
+        "sort": 5,
+        "tagValuesQuery": "",
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      },
+      {
+        "current": {},
+        "datasource": "Metrics",
+        "definition": "",
+        "hide": 2,
+        "includeAll": false,
+        "multi": false,
+        "name": "min_wal_size",
+        "options": [],
+        "query": {
+          "query": "query_result(pg_settings_min_wal_size_bytes{service_name=~\"$service_name\"})",
+          "refId": "Metrics-min_wal_size-Variable-Query"
+        },
+        "refresh": 2,
+        "regex": "/ ([0-9\\.]+)/",
+        "skipUrlSync": false,
+        "sort": 5,
+        "tagValuesQuery": "",
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      },
+      {
+        "current": {},
+        "datasource": "Metrics",
+        "definition": "",
+        "hide": 2,
+        "includeAll": false,
+        "multi": false,
+        "name": "max_parallel_workers_per_gather",
+        "options": [],
+        "query": {
+          "query": "query_result(pg_settings_max_parallel_workers_per_gather{service_name=~\"$service_name\"})",
+          "refId": "Metrics-max_parallel_workers_per_gather-Variable-Query"
+        },
+        "refresh": 2,
+        "regex": "/ ([0-9\\.]+)/",
+        "skipUrlSync": false,
+        "sort": 5,
+        "tagValuesQuery": "",
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      },
+      {
+        "current": {},
+        "datasource": "Metrics",
+        "definition": "",
+        "hide": 2,
+        "includeAll": false,
+        "multi": false,
+        "name": "autovacuum_multixact_freeze_max_age",
+        "options": [],
+        "query": {
+          "query": "query_result(pg_settings_autovacuum_multixact_freeze_max_age{service_name=~\"$service_name\"})",
+          "refId": "Metrics-autovacuum_multixact_freeze_max_age-Variable-Query"
+        },
+        "refresh": 2,
+        "regex": "/ ([0-9\\.]+)/",
+        "skipUrlSync": false,
+        "sort": 5,
+        "tagValuesQuery": "",
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      },
+      {
+        "current": {},
+        "datasource": "Metrics",
+        "definition": "label_values({__name__=~\"pg_up|mysql_up|mongodb_up|proxysql_mysql_status_active_transactions\"}, replication_set)",
+        "hide": 2,
+        "includeAll": true,
+        "label": "Replication Set",
+        "multi": true,
+        "name": "replication_set",
+        "options": [],
+        "query": {
+          "query": "label_values({__name__=~\"pg_up|mysql_up|mongodb_up|proxysql_mysql_status_active_transactions\"}, replication_set)",
```

Comment:
I wonder if querying for anything other than `pg_up` is of relevance here?

Reply:
Great question. I was not using this, and just opted for:

Environment / Cluster / Service Name / Scope Name

However, the `pmm-admin add external` command offers the `--replication-set` argument, so we could add it. The postgresql exporter also has this argument available, so this will boil down to how we want to do it for Postgres in general.

Let me know and I can do the changes.

Reply:
Let's keep it as is for Experimental. 
If this is what I'm thinking - it's the "how to get the list of Replicas in general," and for now, we use the metrics, but when our inventory becomes more reliable ("labels auto discovery") - we must move it to be based on inventory, probably.